docs/advanced/hep_examples.rst

```diff
@@ -0,0 +1,170 @@
+.. _advanced-hep_examples-label:
+
+============
+HEP examples
+============
+
+Compare data and MC
+===================
+
+To make simple data/MC comparison with different MC categories:
+
+.. code-block:: python
+
+    # Define the histograms
+
+    key = "variable_1"
+    range = [-10, 10]
+    category = "category"
+
+    # Define masks
+    signal_mask = df[category] == 7
+    data_mask = df[category] == 8
+
+    background_categories = [0, 1, 2]
+    background_categories_labels = [f"c{i}" for i in background_categories]
+    background_categories_colors = cubehelix_palette(
+        ncolors=len(background_categories),
+        start=1.5,
+        rotation=1.5,
+        darkest=0.3,
+        lightest=0.8,
+        reverse=True,
+    )
+    background_masks = [df[category] == p for p in background_categories]
+
+    # Make histograms
+
+    data_hist = make_hist(df[key][data_mask], bins=50, range=range, weights=1)
```

Comment:
make_hist not defined

Reply:
(I will not comment again for the other examples, but putting the import helps the user to just copy and paste)